packages/babel-cli/package.json

```diff
@@ -29,7 +29,7 @@
     "source-map": "^0.5.0"
   },
   "optionalDependencies": {
-    "chokidar": "^2.1.8"
+    "chokidar": "^3.3.1 || ^2.1.8"
```

Comment:
It will be still resolved to 3.3.1 on node.js 6, which is unsupported. Since `chokidar` is an optional dependency, you can install with `--no-optional` or use resolution to map it to `3.3.1`.

Reply:
> you can install with --no-optional

That will still create a lockfile entry for it which triggers github security alerts.

> use resolution to map it to 3.3.1.

I'm doing that right now.

> It will be still resolved to 3.3.1 on node.js 6, which is unsupported.

Reply:
> Are there plans in the future to closer align with node lts?

Yeah we will bump node requirements to `>= 10.13` on Babel 8: https://github.com/babel/babel/pull/10747